### ISSUE TITLE
adding swr-provider to solid-start router root crashes the ssr  

### ISSUE THREAD

**Issue by crazydaisyew** [tronikelis/solid-swr#63]
Adding the SWRProvider to the root of the ProjectFiles leads to a pnpm error 

 Error when evaluating SSR module /src/entry-server.tsx: failed to import "solid-swr"

Only works if vite config "ssr" rule is disabled

**Comment by tronikelis** [comment]
hey, thanks for the issue report

I am not using ssr myself, so please provide a minimal reproducible example

**Comment by tronikelis** [comment]
maybe this is also related to https://github.com/tronikelis/solid-swr/issues/64 ?

**Comment by tronikelis** [comment]
@crazydaisyew can you check with 5.0.2 ?

**Comment by crazydaisyew** [comment]
@tronikelis nope, still the same, I think it's an internal solid-js issue, maybe something vite config related, i could share the code if it's something you'd want to investigate, i've spend a lot of time trying to find the issue though

**Comment by tronikelis** [comment]
if you could send the code yeah would be nice, I'll try to check it sometime in the future
